sympy/polys/matrices/_dfm.py

```diff
@@ -785,6 +785,50 @@ def lu_solve(self, rhs):
 
         return self._new(sol, sol_shape, self.domain)
 
+    def fflu(self):
+        """
+        Fraction-free LU decomposition of DFM.
+
+        Explanation
+        ===========
+
+        This method computes the PLDU decomposition
+        using Gauss-Bareiss elimination in a fraction-free manner,
+        it ensures that all intermediate results remain in
+        the domain of the input matrix. Unlike standard
+        LU decomposition, which introduces division, this approach
+        avoids fractions, making it particularly suitable
+        for exact arithmetic over integers or polynomials.
+
+        This method satisfies the invariant:
+
+        P * A = L * inv(D) * U
+
+        See Also
+        ========
+
+        sympy.matrices.matrixbase.MatrixBase.LUdecomposition
+        LUdecomposition_Simple
+        LUsolve
```

Comment:
I think that this docstring can just be reduced to the one line summary at the top and then the See Also section at the bottom. In particular the See Also section should point to the DomainMatrix.fflu method which should be the docstring that actually explains what is returned.

The only relevant point that is worth explaining in this docstring is that this will use python-flint if possible for a matrix of integers but otherwise uses the DDM method.